--- index.js.ORIG	Sat Mar 10 05:43:45 2018
+++ index.js	Sat Mar 10 19:23:55 2018
@@ -63,13 +63,13 @@
                 .map(function (bemCell) {
                     var localNamingOpts = levelsMap[bemCell.layer].naming || namingOptions;
                     var fsScheme = levelsMap[bemCell.layer].scheme || 'nested';
-                    var entityPath = path.resolve(bemFs(fsScheme).path(bemCell, localNamingOpts));
+                    var entityPath = path.resolve(bemFs(fsScheme).path(bemCell, localNamingOpts)).replace(/\\/g,'/');
                     // BemFile
                     return {
                         cell: bemCell,
                         exist: fs.existsSync(entityPath),
                         // prepare path for require cause relative returns us string that we couldn't require
-                        path: requiredPath(path.relative(path.dirname(filename), entityPath))
+                      path: requiredPath(path.relative(path.dirname(filename), entityPath)).replace(/\\/g,'/')
                     };
                 });
 
