--- webpack.config.dev.js.ORIG	Sat Mar 10 02:56:48 2018
+++ webpack.config.dev.js	Sat Mar 10 02:57:02 2018
@@ -80,7 +80,7 @@
     // We placed these paths second because we want `node_modules` to "win"
     // if there are any conflicts. This matches Node resolution mechanism.
     // https://github.com/facebookincubator/create-react-app/issues/253
-    modules: ['node_modules', paths.appNodeModules].concat(
+    modules: [paths.appSrc, 'node_modules', paths.appNodeModules].concat(
       // It is guaranteed to exist because we tweak it in `env.js`
       process.env.NODE_PATH.split(path.delimiter).filter(Boolean)
     ),
